tests/test_cli.py

```
@@ -191,14 +200,16 @@ def test_submit_order_unknown_vault(mock_ensure_env, MockOrionConfig, tmp_path):
     order_file = tmp_path / "order.json"
     order_file.write_text('{"0xToken": 1}')
 
-    with patch.dict(os.environ, {"ORION_VAULT_ADDRESS": "0xUnknown"}):
-        result = runner.invoke(
-            app, ["submit-order", "--order-intent-path", str(order_file)]
+    with pytest.raises(
+        ValueError, match="Vault address 0xUnknown not in OrionConfig contract."
```

Ruff (0.14.11)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)